ios/RNSScreenStackHeaderConfig.h

```diff
@@ -17,6 +17,7 @@ typedef NS_ENUM(NSInteger, RNSStatusBarStyle) {
 @property (nonatomic, retain) NSString *title;
 @property (nonatomic, retain) NSString *titleFontFamily;
 @property (nonatomic, retain) NSNumber *titleFontSize;
+@property (nonatomic, retain) NSString *titleFontWeight;
```

Comment:
I think we could add the same but for `largeTitle`, the changes should look pretty similar to the normal title.